<!DOCTYPE html>
<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
      xmlns:th="http://www.thymeleaf.org">
<head>
    <title th:text="${@siteSettings.title + ' - Documentation'}">Documentation</title>
    <object th:include="fragments/head :: head" th:remove="tag"/>
</head>
<body>
<div th:replace="fragments/navbar :: navbar(documentation)">...</div>
<div class="container-fluid p-4">


    <div th:replace="fragments/title :: title(title='About')" th:remove="tag">...</div>

    <div class="main-content p-4">
        <p><b th:text="${@siteSettings.title}">LIST-S2</b> (species independent) predicts the deleteriousness of amino acid mutations in protein sequences. First, it
            aligns (high Local identity Pair-wise Sequence Alignment, LPSA) the query sequence to all protein sequences in the UniProt Swiss-Prot/TrEMBL
            database and then it identifies the most relevant homologies based on their local identity to the query
            sequence around that position. And finally, it estimates the potential deleteriousness
            of mutations based on Taxonomy distance of species with variations to the query.</p>

        <p> Prediction scores are in the range [0 .. 1], where lower scores indicate more benign and higher indicate
            deleteriousness effect. We also included <a th:href="${@siteSettings.precomputedSite}" href="https://precomputed.list-s2.msl.ubc.ca/" target="_blank" rel="noopener noreferrer">precomputed predictions</a>
            for virtually all protein sequences of common taxa identified by their UniProt accession and Ensemble transcriptome id.</p>

    </div>

    <div th:replace="fragments/title :: title(title='Instructions')" th:remove="tag">...</div>

    <div class="main-content p-4">
        <p><b>Input:</b></p>
        <p>One or more protein sequences in fasta format. Optional OX=n can be included in the FASTA header to
            explicitly identify the sequence Taxa ID as n, otherwise, the Taxa ID of the sequence with the highest
            alignment (Bits score) will be used as the query Taxa ID. Taxa IDs can be obtained from
            <a href="https://www.ncbi.nlm.nih.gov/Taxonomy/TaxIdentifier/tax_identifier.cgi" target="_blank"
               rel="noopener noreferrer">NCBI Taxonomy name/id</a></p>
        <p><b>Output:</b></p>
        <ol type="I">
            <li> The potential deleteriousness effect of every possible amino acid mutation in the form of a heat-map
                matrix.
            </li>
            <li> A downloadable text file with the following columns:
                <ol type="1">
                    <li>AC: Sequence id from the fasta header.</li>
                    <li>Pos: Amino acid position.</li>
                    <li>Ref: The reference amino acid.</li>
                    <li>Conservation: The average <span th:text="${@siteSettings.title}" th:remove="tag">LIST-S2</span> deleteriousness score of all possible mutations at that
                        position.
                    </li>
                    <li>20 columns one for each amino acid: The potential deleteriousness <span th:text="${@siteSettings.title}" th:remove="tag">LIST-S2</span> score for mutating the
                        reference amino acid to “this” amino acid.
                    </li>
                </ol>
        </ol>



    </div>

    <div th:replace="fragments/title :: title(title='Terms & Conditions')" th:remove="tag">...</div>

    <div class="main-content p-4" >
        <p>This software is distributed on an as is basis, without any kind of warranties. This software is free for
            academic not for profit research only. Appropriate citation when using <span th:text="${@siteSettings.title}" th:remove="tag">LIST-S2</span> in publication of scientific
            results. </p>
    </div>

</div>


    <div th:replace="fragments/footer" th:remove="tag">...</div>
</body>
</html>
